package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@esri/hub.js",
-  "version": "3.6.5",
+  "version": "3.6.6",
```

Comment:
I think the version number shouldn't be updated until release.

Reply:
@haoliangyu  is correct, the deploy process updates this.

Reply:
good to know.